Meta/CMake/flatpak/org.ladybird.Ladybird.json

```diff
@@ -0,0 +1,464 @@
+      "builddir": false,
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/unicode-org/icu.git",
+          "tag": "release-76-1"
+        }
+      ],
+      "subdir": "icu4c/source/",
+      "build-options": {
+        "cflags": "-fPIC",
+        "cxxflags": "-fPIC",
+        "cppflags": "-DPIC"
+      },
+      "config-opts": [
+        "--prefix=/app",
+        "--libdir=/app/lib",
+        "--disable-samples",
+        "--disable-tests",
+        "--disable-tools",
+        "--disable-extras",
+        "--disable-layoutex",
+        "--disable-debug",
+        "--enable-release",
+        "--disable-shared",
+        "--enable-static",
+        "--with-data-packaging=static"
+      ]
+    },
+    {
+      "name": "vulkan-memory-allocator",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator.git",
+          "tag": "v3.1.0"
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_PREFIX_PATH=/app",
+        "-DCMAKE_INSTALL_LIBDIR=lib"
+      ]
+    },
+    {
+      "name": "gn",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gn.googlesource.com/gn",
+          "commit": "f792b9756418af8ab8a91a4c15b582431cb86ff9"
+        },
+        {
+          "type": "file",
+          "path": "gn-whole-enchilada.sh"
+        }
+      ],
+      "build-commands": [
+        "./gn-whole-enchilada.sh"
+      ]
+    },
+    {
+      "name": "brotli",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/google/brotli.git",
+          "tag": "v1.1.0"
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_PREFIX_PATH=/app",
+        "-DCMAKE_INSTALL_LIBDIR=lib",
+        "-DBROTLI_DISABLE_TESTS=ON",
+        "-DBROTLI_BUILD_TOOLS=OFF"
+      ]
+    },
+    {
+      "name": "libwoff2",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/google/woff2.git",
+          "tag": "v1.0.2",
+          "disable-submodules": true
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_PREFIX_PATH=/app",
+        "-DCMAKE_INSTALL_LIBDIR=lib",
+        "-DCANONICAL_PREFIXES=ON"
+      ]
+    },
+    {
+      "name": "dav1d",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://code.videolan.org/videolan/dav1d.git",
+          "tag": "1.5.1"
+        }
+      ],
+      "config-opts": [
+        "-Denable_tools=false",
+        "-Denable_tests=false",
+        "-Denable_examples=false",
+        "-Denable_asm=true",
+        "-Dbuildtype=release",
+        "-Dlibdir=/app/lib"
```

Comment:
I got the following error trying to build:
```
ERROR: Got argument libdir as both -Dlibdir and --libdir. Pick one.
Error: module dav1d: Child process exited with code 1
```

So I did this and that seemed to work:
```diff
diff --git a/Meta/CMake/flatpak/org.ladybird.Ladybird.json b/Meta/CMake/flatpak/org.ladybird.Ladybird.json
index 158cc650688..b4dc73f0994 100644
--- a/Meta/CMake/flatpak/org.ladybird.Ladybird.json
+++ b/Meta/CMake/flatpak/org.ladybird.Ladybird.json
@@ -264,7 +264,7 @@
         "-Denable_examples=false",
         "-Denable_asm=true",
         "-Dbuildtype=release",
-        "-Dlibdir=/app/lib"
+        "--libdir=/app/lib"
       ]
     },
     {
```

Reply:
Ah, you might have a newer flatpak-builder than me then. Mine didn't pass the libdir to meson automatically, but it did pass the prefix

Reply:
Builder as in the IDE? Last I tried that it only liked at the final sources item of the app module and failed to build. Maybe it's been fixed in the last year or so though